From: Marco Herrn <marco@mherrn.de>

I included a patch that adds a sentence to the manpage for option -bpp,
that currently only 16 bpp are available.

Marco

--- directvnc.1	2003-01-31 08:59:01.000000000 +0100
+++ directvnc.1.new	2004-05-08 18:16:21.000000000 +0200
@@ -41,7 +41,9 @@
 care!
 .TP 5
 .B -b, --bpp
-the bits per pixel to be used by the client
+the bits per pixel to be used by the client.
+Currently only 16 bpp are
+are available.
 .TP 5
 .B -e --encodings
 DirectVNC supports several different compression methods to encode
